src/transaction/log_global.c

```diff
@@ -103,25 +103,32 @@ log_global::~log_global ()
 }
 
 void
-log_global::update_max_ps_flushed_lsa (const LOG_LSA &lsa)
+log_global::update_ps_consensus_flushed_lsa (const LOG_LSA &lsa)
 {
   {
     std::unique_lock<std::mutex> lock (m_ps_lsa_mutex);
-    m_max_ps_flushed_lsa = lsa;
+    m_ps_consensus_flushed_lsa = lsa;
   } // lock.unlock ();
   m_ps_lsa_cv.notify_all ();
 }
 
+log_lsa
+log_global::get_ps_consensus_flushed_lsa ()
```

Comment:
maybe not needed?
or, if needed, restrained to logging needs (eg: assert that some logging sys param is set)
function should also be `const`

Reply:
removed.